api/src/main/proto/stellarstation/api/v1/groundstation.proto

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2018 Infostellar, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package stellarstation.api.v1.groundstation;
+
+option go_package = "stellarstation/groundstation";
+
+option java_multiple_files = true;
+option java_outer_classname = "GroundStationProto";
+option java_package = "com.stellarstation.api.v1.groundstation";
+
+// The public API of StellarStation for use by ground station operators integrating with the
+// StellarStation ground network. It is invalid to specify ground station IDs that the operator
+// does not own. Only using IDs shown on the StellarStation Console or returned in API responses
+// will ensure all inputs are valid.
+//
+// A plan is a scheduled pass that will be executed to send and receive data between the ground
+// station and satellite during the time range.
+//
+// A plan is a scheduled pass that will be executed to send and receive data between the ground
+// station and satellite during the time range.
+service GroundStationService {
+  // Lists the plans for a particular ground station.
+  //
+  // The request will be closed with an `INVALID_ARGUMENT` status if `ground_station_id`,
+  // `aos_after`, or `aos_before` are missing, or the duration between the two times is longer than
+  // 31 days.
+  rpc ListGroundStationPlans (ListGroundStationPlansRequest) returns (ListGroundStationPlansResponse);
```

Comment:
Since this proto is for ground stations, what do you think of naming it as ListPlans?

Reply:
Good idea, done.